app/javascript/react/components/NewReviewForm.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react'
+
+import ErrorList from './ErrorList'
+
+const NewReviewForm = props => {
+  const game_id = props.game_id
+  const [review, setReview] = useState({
+    rating: "",
+    body: "",
+    game_id: game_id
+  })
+  const [errors, setErrors] = useState({})
+  
+  const handleChange = event => {
+    setReview({
+      ...review,
+      [event.currentTarget.id]: event.currentTarget.value
+    })
+  }
+
+  const validateForm = () => {
+    let newErrors = {}
+    let requiredFields = ["rating"]
+    requiredFields.forEach((field) => {
+      if(review[field].trim() === "") {
+        newErrors = {
+          ...newErrors,
+          [field]: "is blank"
+        }
+      }
+    })
+    setErrors(newErrors)
+    return _.isEmpty(newErrors)
+  }
+
+  const handleSubmit = event => {
+    event.preventDefault();
+    if(validateForm()){
+      props.handleFormSubmit(review)
+    }
+  }
+
+  return (
+  <div>
+    <br />
```

Comment:
ideally we should almost never need line break tags, because css could be used instead. just remove this